docs/ecosystem-react.md

```diff
@@ -4,7 +4,7 @@ title: single-spa-react
 sidebar_label: React
 ---
 
-[![Build Status](https://travis-ci.org/single-spa/single-spa-react.svg?branch=master)](https://travis-ci.org/single-spa/single-spa-react)
+[![Build Status](https://travis-ci.com/single-spa/single-spa-react.svg?branch=master)](https://travis-ci.com/single-spa/single-spa-react)
```

Comment:
Using travis-ci.com is now suggested for open source projects. I enabled it for the single-spa github org